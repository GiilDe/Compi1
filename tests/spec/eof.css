"BASIC STRING"
'other basic string'
"
"
