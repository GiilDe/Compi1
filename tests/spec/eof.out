1 STRING BASIC STRING
2 STRING other basic string
Error unclosed string
